### ISSUE TITLE
[Feature Request] Stop All Loops button in settings

### ISSUE THREAD

**Issue by CKY-** [crowbartools/Firebot#1512]
**Describe the solution you'd like**
A button that will force all loops to exit. even if they have not  finished.
If While loops are enabled show a button that  will force all loops to exit. 
This button can set a internal var exit to true and in the loop check for it then break out of any currently running loop. 
This will make not having to restart firebot to rest anything that maybe caused by  the loop effects.

**Comment by CaveMobster** [comment]
I don't think this is something to add to the settings, but we could maybe make this into an effect, or add it to the Clear Effects effect.

**Comment by CKY-** [comment]
sounds good to me.
